action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
```

Comment:
This was a temporary fix for not being able to mock kubectl responses.  My goal is to have a pattern that can be used in other actions as well.